drivers/console/aarch64/console.S

```diff
@@ -95,10 +95,15 @@ endfunc console_getc
 	 * data that hasn't been output. It returns 0
 	 * upon successful completion, otherwise it
 	 * returns -1.
-	 * Clobber list : x0, x1
+	 * Clobber list : x0, x1, x2, x3, x4
 	 * ---------------------------------------------
 	 */
 func console_flush
+#if COREBOOT
+	mov	x4, x30
+	bl	coreboot_cbmc_flush
+	mov	x30, x4
+#endif
```

Comment:
I agree it would be good if we can make this work without adding new source files for each driver, while keeping compatibility for unmodified drivers, including non-upstream ones. I'll wait to see some code for this before commenting further.